lib/service/customer_support_service.dart

```diff
@@ -261,6 +279,67 @@ class CustomerSupportServiceImpl extends CustomerSupportService {
     return await _customerSupportApi.createIssue(payload);
   }
 
+  Future<String> createRenderingIssueReport(
+    AssetToken token,
+    List<String> topics,
+  ) async {
+    /** Generate metadata
+     * Format:
+     * [Platform - Version - HashedAccountDID]
+     * [TokenIndexerID - Topices]
+     */
+    var metadata = "";
+
+    final defaultAccount = await _accountService.getDefaultAccount();
+    final accountDID = await defaultAccount.getAccountDID();
+    final accountHMACSecret =
+        await _configurationService.getAccountHMACSecret();
+
+    final hashedAccountID = Hmac(sha256, utf8.encode(accountHMACSecret))
+        .convert(utf8.encode(accountDID));
+
+    String platform = "";
+    if (Platform.isIOS) {
+      platform = "iOS";
+    } else if (Platform.isAndroid) {
+      platform = "android";
+    }
```

Comment:
```suggestion
    final platform = Platform.isIOS ? "iOS" : "android";
```

Reply:
Thought about it. but I think `else if (Platform.isAndroid)` is more correct then `not Platform.isIOS` (even we only support iOS & Android now).

Reply:
Ah, got it.